src/labelle/lib/devices/device_config.py

```diff
@@ -50,19 +55,57 @@ def matches_device_id(self, idValue: int) -> bool:
         else:
             return False
 
-    def get_tape_print_margins(self, tapeSizeMM: int) -> tuple[int, int]:
+    def get_tape_print_size_and_margins_px(
```

Comment:
Please make this a property, rather a `get_` function.

Reply:
The margins depend on the tape size selected as input.
A property that has a value based on another property's value seems very dirty to me.

I kept the bottom margin for the sake of completeness, they are equal now due to the calculation but that might not always be the case. The current rendering class only uses the top-margin but that could change too...

Reply:
Moved to the labeler class and made it a property / named tuple